packages/backend/src/storage/ChannelsAccessController.ts

```diff
@@ -0,0 +1,61 @@
+import AccessController from 'orbit-db-access-controllers'
+import { getCrypto } from 'pkijs'
+import { stringToArrayBuffer } from 'pvutils'
+import { ChannelMessage, channelMessagesAdapter } from '@quiet/state-manager'
+import { keyObjectFromString, verifySignature } from '@quiet/identity'
+
+const type = 'channelsaccess'
+
+export const createChannelAccessController = (peerId) => {
+  // @ts-ignore
+  class ChannelsAccessController extends AccessController {
+    static get type() {
+      return type
+    }
+
+    async canAppend(entry, identityProvider) {
+      // Channel deletion WIP
```

Comment:
Is this still WIP?

Reply:
We merged this just so that emi's typescript work could move forward?